### PR TITLE
adding the npm npjs-basic project

### PR DESCRIPTION
**Pull request by hrishibawane**
I think my project would be a good fit for Up for Grabs.

Let me know if there is any feedback about the project.

### CODE REVIEW

File: _data/projects/npjs-basic.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+name: npjs-basic
+desc: A numpy-like library for npm
+site: https://example.org/your-project-link-here
+tags:
+  - javascript
+  - node.js
+  - npm
+upforgrabs:
+  name: good first issue, up-for-grabs
```
Comment (shiftkey) [inline _data/projects/npjs-basic.yml]: This needs to be only one label:

```suggestion
  name: up-for-grabs
```

File: _data/projects/npjs-basic.yml [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+  link: https://github.com/hrishibawane/npjs-basic
```
Comment (shiftkey) [inline _data/projects/npjs-basic.yml]: This then becomes the link that corresponds with that label:

```suggestion
  link: https://github.com/hrishibawane/npjs-basic/labels/up-for-grabs
```

**Review by shiftkey — COMMENTED** [review]
Thanks for the contribution @hrishibawane.

Just a couple of changes to get to PR validation passing.

**Comment by hrishibawane** [comment]
Thanks @shiftkey for reviewing. I have committed the required changes for PR validation.

**Review by shiftkey — APPROVED** [review]

**Pull request merged**